README.md

```diff
@@ -72,66 +95,126 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
   mv app/assets/stylesheets/application.css app/assets/stylesheets/application.scss
   ```
 
-1. Delete _all_ Sprockets directives in `application.scss` (`require`, `require_tree` and `require_self`) and use Sass’s native `@import` instead ([why?](http://pivotallabs.com/structure-your-sass-files-with-import)).
+1. Delete _all_ Sprockets directives in `application.scss` (`require`,
+   `require_tree` and `require_self`) and use Sass’s native `@import` instead
+   ([why?][sass-import]).
 
-1. Import Bourbon at the beginning of `application.scss`. All additional stylesheets should be imported below Bourbon:
+1. Import Bourbon at the beginning of `application.scss`. All additional
+   stylesheets should be imported below Bourbon:
```

Comment:
Should or must? Maybe we can be more deliberate here: 

`all other stylesheets using bourbon mixins must be imported below Bourbon` or something that is accurate as to why we are asking them to adjust load order.